CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 BUG FIXES:
 
 * The module installer will now record in its manifest a correct module source URL after normalization when the URL given as input contains both a query string portion and a subdirectory portion. Terraform itself doesn't currently make use of this information and so this is just a cosmetic fix to make the recorded metadata more correct. [GH-31636]
+* Previously, run logs contain the plaintext value of sensitive outputs. With this fix, all sensitive output values will be omitted in the run logs when `terraform apply -json` is run. ([#31813](https://github.com/hashicorp/terraform/issues/31813))
```

Comment:
The term "run logs" isn't commonly understood to Terraform core users, as it's Terraform Cloud specific language.

Some suggestions:

- We should explicitly note that the inclusion of sensitive output values in the `terraform apply -json` was unintentional
- Point users to `terraform output -json` as a way of retrieving machine-readable output values, including those which are sensitive

Reply:
Oh, one other note: I'd generally suggest not editing the CHANGELOG.md file in a PR commit. This commonly causes merge conflicts, most especially with backports to prior releases. The [just-merged pull request template](https://github.com/hashicorp/terraform/blob/main/.github/pull_request_template.md?plain=1) encourages you to use the PR description to propose a changelog entry for reviewers to check out, then manually update the CHANGELOG.md file on the target branch post-merge.